src/core/DataBase_List.h

```diff
@@ -11,64 +11,65 @@
 #include <vector>
 #include <unordered_map>
 
-
 using namespace std;
```

Comment:
Please scope this inside the class declaration, otherwise the global namespace will get polluted.